include/deal.II/numerics/matrix_tools.h

```diff
@@ -68,6 +69,8 @@ namespace PETScWrappers
 #endif
 
 #ifdef DEAL_II_WITH_TRILINOS
+// TODO: move this to a stand-alone header and include from there?
+#  define TrilinosScalar double
 namespace TrilinosWrappers
```

Comment:
Don't bother with ```TrilinosScalar```, I am pretty sure that in some files we use double directly. Epetra only supports double so it doesn't matter anyway. When we will add Tpetra, we can add a typedef TrilinosScalar in ```base/types.h```.